packages/mobile/src/transactions/send.ts

```diff
@@ -189,21 +194,44 @@ export function* wrapSendTransactionWithRetry(
 ) {
   for (let i = 1; i <= TX_NUM_TRIES; i++) {
     try {
-      const { result, timeout, cancel } = yield race({
-        result: call(sendTxMethod),
-        timeout: delay(TX_TIMEOUT * i),
+      const task = yield fork(sendTxMethod)
+      let { result, timeout, cancelled } = yield race({
+        result: join(task),
+        timeout: delay(TX_TIMEOUT * i - TX_TIMEOUT_GRACE_PERIOD),
         ...(cancelAction && {
-          cancel: take(cancelAction),
+          cancelled: take(cancelAction),
         }),
       })
 
+      // In some conditions (e.g. app backgrounding) the app may become suspended, preventing the
+      // send task from making progress. If this occurs for long enough that the timeout elapses, a
+      // timeout may be triggered even if the transaction send is complete. A second race is
+      // triggered here to handle these cases by giving the send task a grace period to return a
+      // result after the initial timeout fires.
```

Comment:
interesting -- ya I think its ok from the functionality perspective. from the user experience, the typically user would return to the "enter message" screen while the logic to make all the transactions would resume progress, so i think pausing it may be more of an optimization.  there could be a decent delay if they background before making the `waitForSelectingIssuers`.

i think then we can close the ticket #5153 and open two more
1. around investigating general transaction suspension
2. optimization for the pausing the countdown timer (would need to sync with product on the priority of this)